### DISCUSSION TITLE
Auto-closing of nodes in the symbol outline?

### DISCUSSION THREAD

**Discussion by pkazmier** [nvim-neo-tree/neo-tree.nvim#1389] [Q&A]
When following the cursor in the symbol outline, nodes open as I move around, but they do not automatically close. Is that expected behavior? I’m trying to find a symbol outline for my environment and was pleasantly surprised to see neotree already had one. But I was hoping the nodes would open automatically **and** close automatically (similar to outline.nvim). Is there a configuration option for this that I’ve overlooked? Thanks!

**Comment by pysan3** [comment]
Unfortunately there is no distinction between explicitly opened nodes and auto opened ones in symbol source. (It is only a feature in filesystem source).

Please open a feature request if you want this feature.

To be precise, do you want a feature to auto close nodes that are not in scope right now (I think this is the behavior of outline.nvim but not sure), or distinguish behavior between manually opened nodes and auto expanded nodes?
